src/sentry/testutils/helpers/backups.py

```diff
@@ -391,6 +393,12 @@ def create_exhaustive_organization(
         OrganizationAccessRequest.objects.create(member=invited, team=team)
 
         # Project*
+        project_template = ProjectTemplate.objects.create(name=f"template-{slug}", organization=org)
```

Comment:
Thank you for updating this!

Overall, this is a great PR vis a vis adding relocation support to a new model - I'll probably use it as a reference when folks ask what to do in the future :)